Tests/DangerTests/GitDiffTests.swift

```diff
@@ -143,8 +168,6 @@ final class GitDiffTests: XCTestCase {
         ]))
     }
 
-    // swiftlint:disable all
-    // swiftformat:disable all
     var testDiff: String {
         """
         diff --git a/.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata b/.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 14...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/GitDiffTests.swift;
  Line: 173;
-->

- :warning: Line should be 140 characters or less: currently 142 characters (`line_length`)